akka-stream-tests/src/test/java/akka/stream/stage/JavaIdentityStage.java

```diff
@@ -0,0 +1,46 @@
+package akka.stream.stage;
+
+import akka.stream.Attributes;
+import akka.stream.FlowShape;
+import akka.stream.Inlet;
+import akka.stream.Outlet;
+
+public class JavaIdentityStage<T> extends GraphStage<FlowShape<T, T>> {
+  private Inlet<T> _in = Inlet.of("Identity.in");
```

Comment:
I prefer "what reads best" in general for Java. APIs like guava have been doing this for years - to make using Java libraries bearable, they need to sound "fluent". Maybe that's a discussion for the entire API though, not this PR :)